backend/databasemodel/model.sql

```diff
@@ -1520,8 +1538,11 @@ CREATE TABLE kooste.museovirastoarcrest_rkykohteet (
 	tarmo_category text DEFAULT 'Nähtävyydet',
 	type_name text DEFAULT 'Rakennettu kulttuurikohde',
 	www text,
+	deleted boolean NOT NULL DEFAULT false,
 	CONSTRAINT museovirastoarcrest_rkykohteet_pk PRIMARY KEY ("OBJECTID")
 );
+CREATE INDEX ON kooste.museovirastoarcrest_rkykohteet (deleted);
+CREATE INDEX ON kooste.tamperewfs_luonnonmuistomerkit (visibility);
```

Comment:
I think there is wrong table name here (kooste.tamperewfs_luonnonmuistomerkit -> kooste.museovirastoarcrest_rkykohteet) :)